apps/editor/src/app/global.less

```diff
@@ -119,43 +119,4 @@
     top: 0;
     background-color: #1675e0;
   }
-}
-
-.react-icons {
```

Comment:
Changes were needed because with switch of icon library we lost the rs-icon class. Readded it to the icons so these changes are not needed anymore